app/services/clamav_service.rb

```diff
@@ -1,6 +1,6 @@
 class ClamavService
   def self.safe_file?(file_path)
-    if Rails.env.development?
+    if Rails.env.development? || !Rails.configuration.x.clamav.enabled
```

Comment:
Si on part sur l'ajout d'une variable d'env, dans ce cas autant l'utiliser même en dévelopment :

```suggestion
    if ENV['CLAMAV_ENABLED'] == 'disabled'
```

(en plus par défaut il est désactivé dans `env.example`, donc pour un développement local ça sera bon par défaut.)